### PR TITLE
[SW-738] Reuse existing ListItemWrapper in all lists with similar styling

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

File: src/popup/pages/FungibleTokens/TokenContainer.vue [hunk 1]
```diff
@@ -227,7 +227,7 @@ export default defineComponent({
 
 .token-container {
   --screen-padding-x: 12px;
-
+  background-color: red;
```
Comment (pstachel) [inline src/popup/pages/FungibleTokens/TokenContainer.vue:230]: Should it be red?

File: src/popup/components/NameRow.vue [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
-    class="name-row"
     @click="$emit('click')"
   >
-    <Avatar
-      :name="name"
-      :address="address"
-    />
-    <div class="name-info">
-      <slot />
+    <div class="name-row">
```
Comment (peronczyk) [inline src/popup/components/NameRow.vue:6]: `name-row` cass should be added to root element of the component.

**Review by peronczyk — APPROVED** [review]

**Review by ifaouibadi — APPROVED** [review]

**Comment by CedrikNikita** [comment]
Commit message must be meaningful.

**Review by CedrikNikita — DISMISSED** [review]
Commit message must be meaningful.

**Comment by Liubov-crypto** [comment]
I see the bigger space between elements in AuctionList than on dev branch: 

спейс

On dev branch: 
вум

**Comment by csiarab** [comment]
Other than the auction page that is different like it was mentioned before i see no other differences compared to dev. The latest transactions look the same now.

**Comment by Liubov-crypto** [comment]
Fixed, LGTM on web and extension

**Pull request merged**